prow/cluster/monitoring/additional-scrape-configs_secret.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: additional-scrape-configs
+  namespace: prow-monitoring
+stringData:
+  prometheus-additional.yaml: |
+    - job_name: k8s-prow-builds-boskos
+      metrics_path: /prometheus
+      static_configs:
+        - targets:
+            - "104.197.27.114"
```

Comment:
Is this the public IP that proxies the `boskos` in the build cluster? ... NVM ... tested with `http://104.197.27.114/prometheus` ... and it seems so. :)

NIP: I hope that we can occupy this IP for good so that no domain is needed.

Reply:
I didn't add this IP, this has been used by Velodrome for ~2 years.